flake.nix

```
@@ -39,17 +163,31 @@
         system:
         let
           pkgs = nixpkgsFor.${system};
-          inherit (self.packages.${system}) niri;
+          niri = pkgs.callPackage niri-package { };
```

```suggestion
          inherit (self.packages.${system}) niri;
```
We don't need to re-evaluate the package

---

o yea. lol. happened 'cause i removed `inherit (self.packages.${system}) niri;` when applying the overlay straightup, and then returned to a different point when undoing that change. nice catch